sphinx_immaterial/apidoc/cpp/api_parser.py

```diff
@@ -164,12 +164,21 @@ class Config:
     but this default is not normally usable, because it will include entities
     defined in the standard library and third-party libraries.
 
+    .. important::
+        Because this is used as a Regular Expression patterns, this should end in
```

Comment:
Perhaps change to:

```
When building on Windows, all path separators are normalized to :python:`"/"`.  Therefore, in the specified regular expressions, always use :python:`"/"` to match a path separator.
```

Reply:
I also changed this for the duplicate admonition (in both `dis/allow_paths`).